docs/phylum.md

```diff
@@ -0,0 +1,37 @@
+---
+title: phylum
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+
+Interact with the Phylum API.
+
+### Core Commands
+* [phylum analyze](https://docs.phylum.io/docs/phylum_analyze)
+* [phylum auth](https://docs.phylum.io/docs/phylum_auth)
+* [phylum project](https://docs.phylum.io/docs/phylum_project)
+
+### Additional Commands
+* [phylum help](https://docs.phylum.io/docs/phylum_help)
+* [phylum history](https://docs.phylum.io/docs/phylum_history)
+* [phylum package](https://docs.phylum.io/docs/phylum_package)
+* [phylum ping](https://docs.phylum.io/docs/phylum_ping)
+* [phylum update](https://docs.phylum.io/docs/phylum_update)
+* [phylum version](https://docs.phylum.io/docs/phylum_version)
+
+### Options
+  `-c`, `--config <FILE>`
+  &emsp; Use a specific configuration file
+  
+  `-h`, `--help`
+  &emsp; Print help information
+  
+  `--no-check-certificate`
+  &emsp; Don't validate the server certificate when performing API requests
+  &emsp; DANGER: this should only be used when SSL termination breaks the Phylum certificate pinning
+  
+  `-t`, `--timeout <TIMEOUT>`
+  &emsp; Set the timeout (in seconds) for requests to the Phylum API
+  
+  `-V`, `--version`
+  &emsp; Print version information
```

Comment:
Based on the way this seems to be formatting, I think I'd suggest using a table instead?

The same applies to all the other places where flags are documented.

Reply:
Markdown renders differently on GitHub, I am going to go ahead and leave this as is for now and see how it renders on our documentation site. If we don't like it, I think a table would work nicely!